tools/cassandra/cqlclient.go

```diff
@@ -93,6 +94,9 @@ var _ schema.DB = (*CqlClient)(nil)
 func NewCQLClient(cfg *CQLClientConfig) (*CqlClient, error) {
 	var err error
 
+	if cfg.ProtoVersion == 0 {
+		cfg.ProtoVersion = cqlProtoVersion
+	}
```

Comment:
I believe the default will be set later. So these three lines can be removed, along with the local constant `cqlProtoVersion`?

Reply:
Good catch. About removing the const, I'm using it in `tools/cassandra/main.go:L108` in this diff, I'm wondering thou if it's a good idea to use `environment.GetCassandraProtoVersion()` (I created that function) in `tools/cassandra/main.go:L108` instead?

Reply:
You already did it in `CASSANDRA_PROTOCOL_VERSION` in tools/cassandra/main.go.
So here just use the constant value. Getting from another env will be overlapping(and conusing)